crates/editor/Cargo.toml

```diff
@@ -28,6 +28,7 @@ test-support = [
 ]
 
 [dependencies]
+chrono = "0.4"
```

Comment:
We already have `chrono` in the workspace dependencies, so we should use `chrono.workspace = true` here.

Could we also keep this list sorted alphabetically?

Reply:
Whoops meant to remove that. Was just printing the time while debugging haha